Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Question/QuestionAC.cs

```diff
@@ -4,7 +4,7 @@ namespace Promact.Trappist.DomainModel.ApplicationClasses.Question
 {
     public class QuestionAC
     {
-        public Models.Question.Question Question { get; set; }
+        public QuestionDetailAC Question { get; set; }
         public SingleMultipleAnswerQuestionAC SingleMultipleAnswerQuestionAC { get; set; }
```

Comment:
please remove AC based naming from variable name. so you do not have to use AC based naming at client side also.